src/cable/server.cr

```diff
@@ -0,0 +1,148 @@
+module Cable
+  def self.server
+    @@server ||= Server.new
+  end
+
+  def self.restart
+    if current_server = @@server
+      current_server.shutdown
+    end
+    @@server = Server.new
+  end
+
+  class Server
+    getter connections
+    getter redis_subscribe
+    getter redis_publish
+    getter fiber_channel
+
+    def initialize
+      @connections = {} of String => Connection
+      @channels = {} of String => Array(Cable::Channel)
+      @redis_subscribe = Redis.new
```

Comment:
When no args are passed here, it's assumed [localhost:6379](https://github.com/stefanwille/crystal-redis/blob/master/src/redis.cr#L95-L98), but if you wanted to use https://openredis.com/ or something on AWS, you'll need to be able to configure this. Maybe we can use Habitat to allow setting up the redis connection, but default to what it is now?

Reply:
good call! :D 

this totally makes sense! let me work on a redis url config